libusb/os/windows_common.h

```diff
@@ -232,12 +232,19 @@ typedef struct USB_DK_TRANSFER_REQUEST {
 	USB_DK_TRANSFER_RESULT Result;
 } USB_DK_TRANSFER_REQUEST, *PUSB_DK_TRANSFER_REQUEST;
 
+enum discovery_status {
+	NO_LONGER_DISCOVERED,
```

Comment:
nit: Maybe comment the meaning of these states (it is not obvious to me, even after reading the code).

Reply:
Thanks a lit for your review. I force-pushed some comments and rename with the goal of being more explicit about this enum and its use. Please let me know whether it is an improvements and whether it is sufficient or not.